test/spec/modules/33acrossBidAdapter_spec.js

```diff
@@ -148,6 +161,12 @@ describe('33acrossBidAdapter:', function () {
       return this;
     };
 
+    this.withDevice = (device) => {
+      merge(ttxRequest, { device });
```

Comment:
well, I made the change mainly because I needed a deep copy of the object, and not just a shallow copy. 
with that  change I am basically avoiding having to define the entire `device: { ... }` structure every time I call `withDevice()`. Each spec calls `withDevice` with only the nested `device` inner fields that are related to the feature under test.

for instance, if I don't do the deep merge, the `returns the viewport dimensions of the top most accessible frame` spec will need to specify all the nested `device` fields:

```
.withDevice({
  ext: {
    ttx: {
      viewport: {
        w: 9876,
        h: 5432,
        availheight: <availheight>,
        maxtouchpoints: <maxtouchpoints>
      }
    }
  },
  w: 728,
  h: 1024,
  pxratio: <pxratio>
})
```
and not just the ones required ` w: 9876, h: 5432,`

Reply:
I think Prebid.js also offers --> `utils.mergeDeep` for a deep merge, so I can use that function instead of lodash merge.